pages/doc/integrations_tas_howto.md

```diff
@@ -168,7 +168,7 @@ Ensure that your environment meet requirements on the Ops Manager side and on th
 * **Ops Manager Requirements**
   VMware Tanzu Observability by Wavefront nozzle has the following requirements:
   * Read-only access to the Doppler Firehose and Cloud Controller.
-  * Access to an Operations for Applications instance and an API token. <!--The API token that you provide and how it is generated depends on whether your Operations for Applications service is onboarded to VMware Cloud services. Service Account API token or server to server app credentials is the recommended approach. See [Managing API tokens](wavefront_api.html#managing-api-tokens).-->
+  * Access to an Operations for Applications instance and authentication credentials. The credential that you provide and how it is generated depends on whether your Operations for Applications service is onboarded to VMware Cloud services. See [Managing API tokens](wavefront_api.html#managing-api-tokens).
```

Comment:
thought: also keep it simple here? we don't need to explain CSP 3 times in one document

Reply:
I would split this to:

* Access to an Operations for Applications instance.
* Authentication credentials. The authentication credentials depend on whether your Operations for Applications service is onboarded to VMware Cloud services.